doc/en/drivers_basic.rst

```diff
@@ -122,7 +122,7 @@ started test process. Have a look at the following code:
 
     plan.add(GTest(
         name='My GTest',
-        driver=BINARY_PATH,
+        executable=BINARY_PATH,
```

Comment:
Calling the parameter 'executable', but the meta var and documentation uses 'binary' most often - this is a bit inconsistent. I would suggest that we call this parameter 'binary' for simplicity. We use 'binary' most often when we need to specify an executable for drivers in testplan2. Consistency makes easy for user.

Reply:
use `binary` instead